### ISSUE TITLE
How to interpret the results

### ISSUE THREAD

**Issue by thestarocean** [junseonghwan/PhylExAnalysis#7]
After running Phylex, there seem to be multiple chain folders, each having several joint trees. How do we determine which of the trees as the final results? Or there is additional step to merge these results?

**Comment by junseonghwan** [comment]
There is a function `FindBestRep` in `PhylExR` package. You just have to specify the root directory containing each of the chains.

Example here: https://github.com/junseonghwan/PhylExAnalysis/blob/5bafad3fb6dc60e4144b18e85bbdba47a9a04fe7/Rscripts/HGSOC_SS3_PhyloPhenoAnalysis.R#L35

**Comment by thestarocean** [comment]
Thank you very much for your help. It seems that joint tree0 from selected chain is the final result since the code stated that:

datum2node <- read. Table(paste(chain_path, "/joint/tree0/datum2node.tsv"

Is that correct?

**Comment by junseonghwan** [comment]
That is correct. `tree0/` contains the corresponds to the tree with the highest likelihood.
